api/tests/integration/scripts/certification/rescore-certifications_test.js

```diff
@@ -0,0 +1,23 @@
+import { main } from '../../../../scripts/certification/rescore-certifications.js';
+import { expect, knex } from '../../../test-helper.js';
+
+describe('Integration | Scripts | Certification | rescore-certfication', function () {
+  it('should save pg boss jobs for each certification course ids', async function () {
+    // given
+    const certificationsCourseIdList = [1, 2];
+
+    // when
+    await main(certificationsCourseIdList);
+
+    // then
+    const [job1, job2] = await knex('pgboss.job').where({ name: 'CertificationRescoringByScriptJob' });
+
```

Comment:
Je ne sais pas si l'ordre est tout le temps garanti par `knex` (donc commentaire peut être inutile), sécuriser le test pour éviter un flaky :

```suggestion
const [job1, job2] = await knex('pgboss.job').where({ name: 'CertificationRescoringByScriptJob' }).orderBy('createdon', 'asc');
```

Reply:
Done